Makefile

```
@@ -102,3 +102,8 @@ update-deps:
 
 .PHONY: update-test-deps
 update-test-deps: update-deps test
+
+.PHONY: integration-test
+integration-test:
+	go build ./cmd/dao_tests/pubkey_test.go
```

I think I've misslead you, the proper line to take example from IB was the next line:
```suggestion
	go test -c -tags=integration -o dao_tests.test ./cmd/dao_tests
```